src/renderer/components/QrReader/index.tsx

```diff
@@ -0,0 +1,484 @@
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react'
+import {
+  ZBarConfigType,
+  ZBarScanner,
+  ZBarSymbol,
+  ZBarSymbolType,
+  getDefaultScanner,
+  scanImageData,
+} from '@undecaf/zbar-wasm'
+import classNames from 'classnames'
+import { Spinner } from '@blueprintjs/core'
```

Comment:
please don't use blueprint in new code

Reply:
I wouldn't like to write a new spinner as part of this PR, do you have any suggestions what do do instead?

Reply:
take one from https://cssloaders.github.io and mention where it comes from to comply with the MIT licence

Reply:
Is it okay to do this in a subsequent PR? To me it feels like it's own thing

Reply:
we can do that, if we don't forget it.